modules/atomic-widgets/atomic-controls/types/textarea-control.php

```diff
@@ -0,0 +1,32 @@
+<?php
+namespace Elementor\Modules\AtomicWidgets\AtomicControls\Types;
+
+use Elementor\Modules\AtomicWidgets\Base\Atomic_Control_Base;
+
+if ( ! defined( 'ABSPATH' ) ) {
+	exit; // Exit if accessed directly
+}
+
+class Textarea_Control extends Atomic_Control_Base {
+	private string $placeholder = '';
```

Comment:
maybe should be undefined if its empty?

Reply:
I will change this class to only hold the props and to set the placeholder in the props